processor/otel/metrics.go

```diff
@@ -157,6 +179,23 @@ func (c *Consumer) addMetric(metric pdata.Metric, ms *metricsets) bool {
 	return false
 }
 
+func histogramSample(metricName string, bucketCounts []uint64, explicitBounds []float64) model.Sample {
+	values := make([]float64, 0, len(explicitBounds))
+	counts := make([]int64, 0, len(bucketCounts))
+	for i, count := range bucketCounts {
+		if count > 0 {
+			counts = append(counts, int64(count))
+			values = append(values, explicitBounds[i])
```

Comment:
from the otel code:
> 	// The number of elements in bucket_counts array must be by one greater than
	// the number of elements in explicit_bounds array.

Wouldn't that lead to an index-out-of-bound exception for `explicitBounds[i]`?

Reply:
The tests should also be updated if this is the case, as the slices representing `bucket_counts` and `explicit_bounds` are equal length